### PR TITLE
Tanvir dev

### PR DESCRIPTION
**Pull request by Tanvir-Rahman123**
Final merge for protoype 1. Features include: Login, Register, Upload, Home Page Layout, Profile Page Layout

### CODE REVIEW

**Review by R313mon — APPROVED** [review]

File: emuu-client/src/components/UserAuthentication/RegisterScreen.js [hunk 1]
```diff
@@ -27,45 +31,84 @@ function Register() {
...
+       </div>
        <br />
-       <input type= "password" value = {password} onChange ={(e) =>setPassword(e.target.value)} className= "form-control" placeholder = "Password" />
+
+        {/*Password */}
+       <div className="col-sm-6 offset-sm-3">
+       <input type= "password" value = {password} onChange ={(e) =>setPassword(e.target.value)} className= "form-control" placeholder = "Password" required/>
+       <div class="valid-feedback">Valid.</div>
+       <div class="invalid-feedback">Password must be at least 6 characters long!</div>
+       </div>
```
Comment (Nabeel-Asghar) [inline emuu-client/src/components/UserAuthentication/RegisterScreen.js:103]: I don't understand why you're making your own text inputs. Every component library as built in tools for this

File: emuu-client/src/components/home/Home.js [hunk 2]
```diff
@@ -19,18 +21,66 @@ const theme = createTheme({
...
+                     </div>
+                    <div className = "spacer">
+                     <iframe width="382" height="215"
+                       src="https://www.youtube.com/embed/4XmfNkB8HUY"
+                        title="YouTube video player" frameborder="0"
+                      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                       allowfullscreen></iframe>
+                     </div>
+                     </p>
+
```
Comment (Nabeel-Asghar) [inline emuu-client/src/components/home/Home.js:83]: Using divs and ps and h2s is not good

File: emuu-client/src/components/home/Home.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+.break {
+    width: 200px;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
+}
+.gap {
+
+				height:70px;
+			}
```
Comment (Nabeel-Asghar) [inline emuu-client/src/components/home/Home.scss:16]: what is this spacing?

File: emuu-server/firebase/middleware.go [hunk 4]
```diff
@@ -8,10 +8,18 @@ import (
...
+// w.Header().Set("Access-Control-Allow-Credentials", "true")
+// w.Header().Set("Access-Control-Allow-Methods", "GET; POST; OPTIONS")
+// }
 const (
 	authorizationHeader = "Authorization"
-	apiKeyHeader        = "X-API-Key"
-	cronExecutedHeader  = "x-Appengine-Cron"
+// 	apiKeyHeader        = "X-API-Key"
+// 	cronExecutedHeader  = "x-Appengine-Cron"
 	valName             = "FIREBASE_ID_TOKEN"
```
Comment (Nabeel-Asghar) [inline emuu-server/firebase/middleware.go:23]: commented code should never go into main

File: emuu-server/main/main.go [hunk 5]
```diff
@@ -25,13 +29,13 @@ func main() {
...
 		auth.POST("/register", register.CreateUser)
-       // auth.POST("/login", login.LoginUser)
+
 	}
 
 	api := r.Group("api").Use(firebaseSer.AuthJWT)//create a new router with the middleware authJWT
 	{//you should supply the jwt token from firebase
 		api.POST("/", func(ctx *gin.Context) {
-			log.Println("Hello")
+			fmt.Println("Hello")
```
Comment (Nabeel-Asghar) [inline emuu-server/main/main.go:38]: print statements should never go into main

File: emuu-server/users/register.go [hunk 6]
```diff
@@ -11,14 +11,14 @@ import (
...
-	w.Header().Set("Access-Control-Allow-Methods", "GET; POST; OPTIONS")
-}*/
+// func corsAcceptance(w http.ResponseWriter, r *http.Request) {
+// 	w.Header().Set("Content-Type", "application/json")
+// 	w.Header().Set("Access-Control-Allow-Origin", "http://localhost:3000/Register")
+// 	w.Header().Set("Access-Control-Max-Age", "15")
+// 	w.Header().Set("Access-Control-Allow-Origin", "*")
+// 	w.Header().Set("Access-Control-Allow-Credentials", "true")
+// 	w.Header().Set("Access-Control-Allow-Methods", "GET; POST; OPTIONS")
+// }
```
Comment (Nabeel-Asghar) [inline emuu-server/users/register.go:21]: here too

**Comment by Nabeel-Asghar** [comment]
@Tanvir-Rahman123 @R313mon @Moamin1 please look at these comments. Just because it's the first prototype doesn't mean it has to have rough code. I also want you guys to use Prettier to format the javascript before you open up another PR. 

Also, I see material UI imported but never used at all. Why waste time making buttons and forms when this is already built out?

**Pull request merged**